docs/skopeo.1.md

```diff
@@ -29,10 +33,20 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   By default, uses the authorization state in `$HOME/.docker/config.json`,
   which is set e.g. using `(docker login)`.
 
+  **docker-archive:**_docker-reference_
+  An image _tag_ stored in the local docker archives
+
+  **docker-daemon:**_docker-reference_
+  An image _tag_ stored in the docker daemon internal storage
```

Comment:
`s/_tag_/_docker-reference_/`. Also, `_docker-reference_` must contain either a tag or a digest.

Alternatively, when reading images, the format can also be **docker-daemon:**_algo_`:`_digest_ (an image ID).